coralproject/talk
# Confirmation Dialog for Featuring Comments on the Admin Side

## What does this PR do?
- Adds a Confirmation Dialog for Featuring Comments on the Admin Side

## How do I test this PR?
- Write a comment
- Go to the Admin Panel
- Go to Mod
- Feature a comment
- A dialog should pop up
